From 5f843f7c8554e2feab7f57d8718231408196fd80 Mon Sep 17 00:00:00 2001
From: Michael Weiss <dev.primeos@gmail.com>
Date: Fri, 1 Nov 2019 18:47:14 +0100
Subject: [PATCH] setup.py: Fix the setuptools version check

This broke the Nix build after setuptools.__version__ changed from
"41.2.0" to "41.4.0.post20191022".
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0d987d5c..4f625d7c 100644
--- a/setup.py
+++ b/setup.py
@@ -63,7 +63,7 @@ try:
 
     setup, Command = setuptools.setup, setuptools.Command
 
-    observed_version = [int(x) for x in setuptools.__version__.split('.')]
+    observed_version = [int(x) for x in setuptools.__version__.split('.')[0:3]]
     required_version = [36, 2, 0]
 
     # NOTE(etingof): require fresh setuptools to build proper wheels
-- 
2.23.0

